### ISSUE TITLE
JAWS drops out of forms mode when focus is programatically moved to an <input> on ArrowUp or ArrowDown

### ISSUE THREAD

**Issue by Epic-Third-Party-Bug-Reporting** [FreedomScientific/VFO-standards-support#883]
### Summary
If focus is programmatically moved to an <input> when the user presses ArrowUp or ArrowDown, JAWS switches to browse mode.
This prevents continued arrowkey navigation and prevents immediate entry into the `<input>`.

[Reproducible Codepen.](https://codepen.io/Kaulabau/pen/KwPRJLW)
### Expected result
JAWS should stay in forms mode
### Actual result
JAWS switched to browse mode
### Version Info
JAWS: Reproduced on both 2024.2411.16 and 2025.2412.50. Was not present with JAWS 2022, didn't test with 2023.
Browsers: Reproduced on Chrome 131.0.6778.266 and Firefox 134.0.1.
OS: Windows 11, version 24H2.

**Comment by JAWS-test** [comment]
This is an error caused by the automatic form mode change in JAWS. This should be avoided because it is so error-prone, see e.g. 

* https://github.com/FreedomScientific/standards-support/issues/614
* https://github.com/FreedomScientific/standards-support/issues/688
* https://github.com/FreedomScientific/standards-support/issues/679
* https://github.com/FreedomScientific/standards-support/issues/465
* https://github.com/FreedomScientific/standards-support/issues/690
* https://github.com/FreedomScientific/standards-support/issues/803

**Comment by JeremiahRogersVispero** [comment]
This is a scenario where `role="application"` should be used on a container of the text input. With a traditional `<input>` or `<textarea>`, the user should expect that using the arrow keys will move the cursor among text within the input and, in the case of JAWS running, that when there is no more text to traverse, JAWS will manage the state of forms mode for them. If arrow keys are doing something other than their standard behavior, a `role="application"` on the container including the text input should cause JAWS to honor customized arrow behavior like that described in the posting.
